lib/Core/Executor.cpp

```diff
@@ -942,6 +879,51 @@ void Executor::branch(ExecutionState &state,
       addConstraint(*result[i], conditions[i]);
 }
 
+std::tuple<bool, uint64_t> getUINTOperandFromMDFirstAnnotInBB(Instruction* anyInstr, StringRef kind, bool bAssert) {
+        for (Instruction& instr : anyInstr->getParent()->getInstList()) {
+                MDNode* md = instr.getMetadata(kind);
+                if (md) return std::make_tuple(true, mdconst::dyn_extract<ConstantInt>(md->getOperand(0))->getZExtValue());
+        }
+        errs() << "getUINTOperandFromMDFirstAnnotInBB: --no instrumentation--\n";
+        return std::make_tuple(false,0);
+}
+
+void Executor::add_forked_state_to_concolic_priority_list(ExecutionState* otherState){
+	Instruction* lastInstr = otherState->prevPC->inst;
```

Comment:
if we want to have some prioritization for seed scheduling, we should add our own searcher in https://github.com/klee/klee/blob/master/lib/Core/Searcher.cpp